### PR TITLE
[full-ci] update reva, wopi folderurl

### PR DESCRIPTION
**Pull request by micbar**
## Description

Make the wopi folder url of the WOPI App driver configurable.

## Motivation and Context
Provide a meaningful landing page after exiting the Web Editor

@kulmann This currently uses a template to create a Web Url

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: services/app-provider/pkg/config/defaults/defaultconfig.go [hunk 1]
```diff
@@ -33,7 +33,10 @@ func DefaultConfig() *config.Config {
 		},
 		Driver: "",
 		Drivers: config.Drivers{
-			WOPI: config.WOPIDriver{},
+			WOPI: config.WOPIDriver{
+				WopiFolderURLBaseURL:      "https://localhost:9200/",
```
Comment (micbar) [inline services/app-provider/pkg/config/defaults/defaultconfig.go:37]: Should we leave that empty? I want the `OCIS_URL` to be used if set. @kobergj

**Comment by micbar** [comment]
@mmattel Ping for new ENV variables review.

File: services/app-provider/pkg/config/defaults/defaultconfig.go (hunk 1 above)
Comment (kobergj) [reply services/app-provider/pkg/config/defaults/defaultconfig.go:37]: No you can leave it in. You should be able to overwrite it by setting `OCIS_URL` or `APP_PROVIDER_WOPI_FOLDER_URL_BASE_URL`

**Comment by butonic** [comment]
I don't see a problem with this PR but it seems something broke with the reva update

**Comment by ScharfViktor** [comment]
I tried to test it and here my notes:
- if I start docker-compose  https://github.com/owncloud/ocis/blob/master/deployments/examples/ocis_wopi/docker-compose.yml without `APP_PROVIDER_WOPI_FOLDER_URL_BASE_URL` env then after clicking `open file location` I go to `https://localhost:9200/f/1284d238-aa92-42ce-bdc4-0b0000009157$ee6c8003-363a-46f2-86c4-c9e7b319d497!0bbacaeb-a036-46bf-8f62-6d80e5bbb8aa`

https://user-images.githubusercontent.com/84779829/194043177-bdb4ce66-94c1-4440-8581-e78fadccb88f.mov



- probably we should add env`APP_PROVIDER_WOPI_FOLDER_URL_BASE_URL: https://${OCIS_DOMAIN:-ocis.owncloud.test}` to [ocis-appdriver-onlyoffice ](https://github.com/owncloud/ocis/blob/master/deployments/examples/ocis_wopi/docker-compose.yml#L126)? 
- looks better in this case, but we have no file(bug?)

https://user-images.githubusercontent.com/84779829/194043901-d9cd430e-be59-402a-a1bb-b94ecc51288b.mov



- and I would suggest to up onlyoffice version to `onlyoffice/documentserver:7.2`

**Pull request merged**